actionview/lib/action_view/helpers/url_helper.rb

```diff
@@ -3,7 +3,6 @@
 require "active_support/core_ext/array/access"
 require "active_support/core_ext/hash/keys"
 require "active_support/core_ext/string/output_safety"
-require "action_view/helpers/content_exfiltration_prevention_helper"
```

Comment:
As the Urlhelper now uses `form_tag` I think we should probably require 'form_tag_helper':
```suggestion
require "action_view/helpers/form_tag_helper"
```

Reply:
An alternative I’ve been thing about is moving button_to to FormTagHelper.

Reply:
Also, UrlHelper defines low level methods like url_for, which is used by higher level form helpers, and high level methods like button_to which builds on form helpers. Maybe link_to and button_to should be moved to something like NavigationHelper?

Reply:
@rafaelfranca would re-arranging methods defined by Action View Helpers be a breaking change?